test/Riok.Mapperly.Tests/Mapping/UserMethodTest.cs

```diff
@@ -72,6 +72,68 @@ public void WithExistingInstance()
             .Be("target.StringValue = source.StringValue;");
     }
 
+    [Fact]
+    public void NullableToNonNullableShouldThrow()
```

Comment:
IMO this should be extracted into a separate class `NullableTest`, as this doesn't have anything to do with user methods.

Reply:
Moved.